src/TestCZT.jl

```diff
@@ -0,0 +1,17 @@
+using PointSpreadFunctions
+using Revise
```

Comment:
Revise should not be part of this package. There should be no new dependency!
If you use revise, put it into your base environment, but not here!